run-wasm/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "run-wasm"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+
```

```suggestion
publish = false
```

We should also ensure this is consistent with the other example (i.e. changing the winit example to use `edition.workspace`)